infrastructure/lookerstudio/lookerstudio-deployment-walkthrough.md

```diff
@@ -0,0 +1,89 @@
+## Marketing Data Engine Looker Studio Deployment Walkthrough
+
+These steps will generate the URL to deploy the Looker Studio Marketing Data Engine dashboard.
+
+## Environment Set-Up
+
+The installation script's library requirements are minimal however if you wish, you can configure a virtual environment.
+
+```sh
+cd infrastructure/lookerstudio
+python -m venv mde
```

Comment:
Use poetry instead of venv, that is the standard we all are following.

Reply:
Done as part of refactor.